src/parameterizations/lateral/MOM_internal_tides.F90

```diff
@@ -2892,7 +2905,13 @@ subroutine PPM_reconstruction_y(h_in, h_l, h_r, G, LB, simple_2nd)
     enddo ; enddo
   endif
 
-  call PPM_limit_pos(h_in, h_l, h_r, 0.0, G, isl, iel, jsl, jel)
+  select case(trim(adv_limiter))
+    case ("positive")
```

Comment:
String comparison during run-time can be more expensive than comparing an encoding integer value.  We might want to consider revising this code to encode the advection limiter as an integer, as is done in other modules, like MOM_CoriolisAdv.F90.